source/System.h

```diff
@@ -164,6 +164,8 @@ class System {
 	// in per frame).
 	double Danger() const;
 
+	void UpdateRandomLinks(System &previousSystem);
+
 
```

Comment:
Should be a private method.

Reply:
if its going to be private it loses its use which is to nitpick what we redefine
so I made it a part of updateneighbors